docs/user-guide/src/ipam/ipam_installation.md

```diff
@@ -0,0 +1,56 @@
+# Installing IPAM as Deployment
+
+This section will show how  IMAP can be installed as a deployment in a cluster. We will discuss the deployment worklow of IPAM.
+
+## Deploying controllers
+
+CAPI and IPAM controllers need to be deployed at the begining. The IPAM controller has a dependency on Cluster API *Cluster* objects. CAPI CRDs and controllers must be deployed and the cluster objects should exist for
+successful deployments.
+
+## Deployment
+
+The user can create the **IPPool** object independently. It will wait for its cluster to exist before reconciling. If the user wants to create **IPAddress** objects manually, they should be created before any claims. It is highly recommended to use the *preAllocations* field itself or have the reconciliation paused.
+
+After an **IPClaim** object creation, the controller will list all existing **IPAddress** objects. It will then select randomly an address that has not been allocated yet and is not in the *preAllocations* map. It will then create an **IPAddress** object containing the references to the **IPPool** and **IPClaim** and the address, the prefix from the address pool or the default prefix, and the gateway from the address pool or the default gateway.
+
+### Deploy IPAM
+
+Deploys IPAM CRDs and deploys IPAM controllers
```

Comment:
Can you please mention that you are in the IPAM git repo and running Makefile targets inside it, at least at the beginning?

```suggestion
Deploys IPAM CRDs and IPAM controllers
```